Dockerfile

```diff
@@ -0,0 +1,35 @@
+## golang image created 2021-06-24T00:31:06.02014601Z 
+FROM docker.io/library/golang@sha256:be99fa59acd78bb22a41bbc1e15ebfab2262498ee0c2e28c3d09bc44d51d1774 AS builder
+WORKDIR /go/src/github.com/komish/preflight
```

Comment:
I am just thinking if we move the code to the openshift repo, this needs to be updated. Is there anyway to customize the `WORKDIR`

Reply:
This is an arbitrary directory for building the preflight binary and it does not make it to the final image. It really could be named anything to be honest. My thoughts are that we can just change this when the repo moves.

Reply:
we can do a build variable  - https://vsupalov.com/docker-env-vars/

value can be taken from Makefile: `$(IMAGE_BUILDER) build -build-arg s WORKDIR=/go/src/github.com/komish/preflight -t $(IMAGE_REPO)/preflight:$(VERSION) . `

however in that case note the behavior seems different with docker build vs. podman build https://github.com/containers/buildah/issues/2888

Reply:
This directory is for the build stage and does not make it into the resulting image or get used in any way, so to me using a variable is adding complexity that is not really needed. The directory could be called `foo` and it really would not matter at the end.  :)  I could just change it to `preflight` or something if this is that big of a deal.

Reply:
I changed the name of the directory used in the build stage to `go/src/preflight`